HPL.dat

```diff
@@ -0,0 +1,31 @@
+HPLinpack benchmark input file
```

Comment:
Do we need to add it? It's out of any folder.

Reply:
Removed